packages/f-checkout/src/components/Checkout.vue

```diff
@@ -22,6 +22,14 @@
                 type="post"
                 :class="$style['c-checkout-form']"
                 @submit.prevent="onFormSubmit">
+                <p
```

Comment:
QUESTION: Do we think it would be a good idea to try and extract some of the markup out of this component into smaller dumb components?

I don't mean to reuse between registration and checkout, but a small component within the F-Checkout package just to keep the main component concise and readable.

Just a question because I wonder the same thing in F-Registration where our main component ends up quite large

Reply:
I think a logout/login component is a good idea.

Reply:
Agree here.